contributivity_measures.py

```diff
@@ -19,9 +19,27 @@
 import shapley_value.shapley as sv
 
 
-# Contributivity measures functions
+# Compute independent performance scores of models trained independently on each node
+    
+def compute_independent_scores(node_list, collaborative_score):
+    
+    print('\n### Launching computation of perf. scores of models trained independently on each node')
+    
+    # Initialize a list of performance scores
+    performance_scores = []
+    
+    # Train models independently on each node and append perf. score to list of perf. scores
+    for node in node_list:
+        performance_scores.append(fl_train_eval.single_train_score(node)[1])
```

Comment:
Where does the `[1]` come from ?

Reply:
`single_train_score` and `fl_train_score` return the loss ([0]) and the performance ([1])